### PR TITLE
Renaming macros to fix build warning.

### PR DESCRIPTION
**Pull request by leaveleafs**

### CODE REVIEW

**Comment by leaveleafs** [comment]
@lwdupont @kristinathai

File: INTUGroupedArray.podspec [hunk 1]
```diff
@@ -1,10 +1,10 @@
 Pod::Spec.new do |s|
   s.name                  = "INTUGroupedArray"
-  s.version               = "1.1.4"
+  s.version               = "1.1.5"
```
Comment (lwdupont) [inline INTUGroupedArray.podspec:3]: Thanks for bumping this :)

**Comment by lwdupont** [comment]
Looks good to me. 👍

**Comment by lwdupont** [comment]
I believe I released this to cocopods.org too, just a few minutes ago.

**Pull request merged**